tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
@mikeller I guess the main question that I've got is how can we move this forward? I've got tons of ideas on how to put current mess into a structure, but we need to be on the same page in terms of coding approach & style. I'm currently adding a grid framework to the DOM, trying to move it to reusable components that you don't need to redefine for every tab (as seems to be the case with current code).

Also I'm quite surprised that there's non-betaflight support. I think it should be dropped in order not to create config tool competition, allowing for cleaner code base that's specific for Betaflight.